code/modules/client/verbs/who.dm

```diff
@@ -63,7 +63,8 @@
 	var/msg = "<b>Current Admins:</b>\n"
 	if(holder)
 		for(var/client/C in GLOB.permissions.admins)
-			msg += "\t[C] is a [C.holder.rank_name()]"
+			var/improper_rank = "\improper[C.holder.rank_name()]"
+			msg += "\t[C] is \a [C.holder.rank_name()]"
```

Comment:
Not using the variable you just made

Reply:
Fuck, I saw this when turdis flagged it, and didn't push the fix, please hold